server/models/users.js

```diff
@@ -0,0 +1,28 @@
+var thinky = require('../config/database');
+
+var r = thinky.r;
+var type = thinky.type;
+
+// Create the model
+var User = thinky.createModel("users", {
+  id: type.string(),
+  name: type.string(),
+  email: type.string()
+});
+
+// Define User API Endpoints
+var UserApi = {
+  getAll: function(req, res){
```

Comment:
Agree, Separation of concern. This https://github.com/d4ncer/nugget/commit/715dc18baa50f7c471c614615198b39884aecb72 fixes that.